src/views/Package/components/PackageDocsError/PackageDocsError.tsx

```diff
@@ -20,11 +20,12 @@ export const PackageDocsError: FunctionComponent<PackageDocsErrorProps> = ({
       align="center"
       fontSize="xl"
       fontStyle="oblique"
+      px={4}
       wordBreak="break-word"
     >
       Documentation in {language} is still not ready for this package. Come back
-      soon. If this issue persists, please let us know by creting an {issueLink}
-      .
+      soon. If this issue persists, please let us know by creating an{" "}
```

Comment:
Is the additional {" "} intentional?

Reply:
Yep, this is to preserve the space between the text & the `<ExternalLink />` on the next line!